docs/platform/SEPES component overview.md

```diff
@@ -0,0 +1,121 @@
+# SEPES #
+
+In this document SEPES, with its different components, will be described/ documented. 
+
+## SEPES Frontend ##
+
+Frontend allowes users to manage 
+
+This contains the following roles:
+- Admin: Access to all resources 
+- Sponsors: Responsible for sandbox management 
+- Supplier: External vendors
+
+Tech Stack: 
+
+Maturity: 3
+
+Protocol: https
+
+## Azure Portal ##
+
+Azure Portal is used by vendors to create resources for their experiment. 
+Documentation can be found at microsoft homepage
+
+Tech Stack: N/A
+
+Maturity: 
+
+Protocol: https
+
+## SEPES REST API (Backend) ##
+
+Coordinate the different services, and make actions on the behalf of the user.
+
+Tech Stack: asp.net core, C#
+
+Maturity:
+
+Protocol: Uses https rest protocol and can be configured to use http. http should only be used for cases that requires it.
+
+As an example docker compose is configured to use http, this is still safe as the docker wrapper uses https outside.
```

Comment:
Its only safe its behind a proxy that handle HTTPS terminations. Or in local development setting. Docker compose have nothing to do with protocol on it own. Proxy is the important keyword.